### ISSUE TITLE
FA icons not loading on the page / showing as square

### ISSUE THREAD

**Issue by odetoi** [FortAwesome/Font-Awesome#17543]
I have reasearched this online, but can't find a fix that works. 

Here's the url: https://healthwellnessdoctor.com/
The icon is in the footer. 

- Hosting SiteGround. 
- Using E Pro page builder with Hello theme. 
- Have selected 'Yes' for load Font Awesome 4 Support. 
- Have turned host caching and plugin caching off and tested 
- Turned off all plugins except Elementor and E Pro and tested. 
- Cleared browser and website cache.
- Core, plugins and theme are up to date. 

Thanks.

**Comment by MarneeDear** [comment]
I am having a similar problem. I think it is the certificate on fontawesome.com. It was revoked.

Firefox says:

```
An error occurred during a connection to fontawesome.com. Peer’s Certificate has been revoked.

Error code: SEC_ERROR_REVOKED_CERTIFICATE
```

I pull fontawsome like this:

`<script src="https://kit.fontawesome.com/7a2492243c.js" crossorigin="anonymous"></script>`

**Comment by tagliala** [comment]
@MarneeDear any chance that you are using Firefox Nightly?

**Comment by tagliala** [comment]
Hi @odetoi , 

Thanks for being part of the Font Awesome Community.

I can see that you are loading Font Awesome through a third-party plugin (elementor).

We do not provide support for third party plugins

There are also some stylesheets properties preventing icon to display correctly, I'm referring to:

```
.elementor-kit-242 a {
    color: #161616;
    font-family: "Fira Sans", Sans-serif;
    font-size: 1em; /* This property */
    font-weight: 600; /* This property */
    text-transform: none;
    line-height: 1.417em;
    letter-spacing: 0px;
}
```

We have an official wordpress plugin, which is the only one supported: https://fontawesome.com/how-to-use/on-the-web/using-with/wordpress

Hope it helps

Closing here

**Comment by MarneeDear** [comment]
> 
> 
> @MarneeDear any chance that you are using Firefox Nightly?

Yes, I am using Firefox Nightly. I will try in Chrome.

**Comment by tagliala** [comment]
Hi, it looks it was related to Firefox nightly and should have been fixed: FortAwesome/Font-Awesome#17535
